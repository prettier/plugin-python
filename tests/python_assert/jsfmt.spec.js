run_spec(__dirname, ["python"], "*");
