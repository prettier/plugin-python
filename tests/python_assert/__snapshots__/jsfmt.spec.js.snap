// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assert.py 1`] = `
assert 3 + 3
assert False, 'message'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
assert 3 + 3
assert False, "message"

`;

exports[`assert.py 2`] = `
assert 3 + 3
assert False, 'message'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
assert 3 + 3
assert False, "message"

`;
